src/components/AdvanceAnalyticsV2/tabs/AnalyticsTable.jsx

```diff
@@ -29,6 +32,12 @@ const AnalyticsTable = ({
     endDate,
     // pages index from 1 in backend, frontend components index from 0
     currentPage: currentPage + 1,
+    pageSize,
+  });
+
+  const CSVDownloadURL = EnterpriseDataApiService.getEnrollmentsCSVDownloadURL(enterpriseId, {
+    start_date: startDate,
+    end_date: endDate,
```

Comment:
`AnalyticsTables` is a common component being used for all data tables in V2. But here we are fetching data for enrollments. I think the changes should be generic to cater all data tables.